### PR TITLE
Follow-up to Added troubleshooting and e2e info for persistent storage (PR#1609)

### PR DESCRIPTION
**Pull request by ahardin-rh**
Follow-up to https://github.com/openshift/openshift-docs/pull/1609

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@adellape @tnguyen-rh @bfallonf @tpoitras  PTAL

File: install_config/storage_examples/shared_storage.adoc [hunk 1]
```diff
@@ -11,23 +11,27 @@
...
+
+For an explanation of persistent volumes (PVs), persistent volume claims (PVCs),
+and using NFS as persistent storage, see the section on
 link:../persistent_storage/persistent_storage_nfs[Persistent Storage Using NFS].
-This topic shows an end-to-end example of using an existing NFS cluster as an OpenShift persistent store.
-It is assumed that a working NFS server and exports have already been setup. 
-
+This topic shows an end-to-end example of using an existing NFS cluster as an
+{product-title} persistent store. It is assumed that a working NFS server and
+exports have already been setup.
```
Comment (bfallonf) [inline install_config/storage_examples/shared_storage.adoc]: @ahardin-rh I know this was already here when you got here, but the last sentence might need rewording 'cos expletive and "setup" is a typo. Maybe something like:

"This topic shows and end-to-end example of using an existing NFS cluster and {product-title} persistent store, and assumes an existing NFS server and exports exist in your {product-title} infrastructure." 

Though I'm not sure what 'exports' are...

File: install_config/storage_examples/shared_storage.adoc [hunk 2]
```diff
@@ -363,9 +368,10 @@ status:
 <4> The PVC name used by the pod.
 ====
 
-== Creating an additional pod to reference the same PVC
-This pod defintion, created in the same namespace, uses a different container. However, we can use the 
-same backing storage by specifying the claim name in the volumes section below:
+== Creating an Additional Pod to Reference the Same PVC
+This pod definition, created in the same namespace, uses a different container.
+However, we can use the  same backing storage by specifying the claim name in
```
Comment (bfallonf) [inline install_config/storage_examples/shared_storage.adoc]: Double space above

**Comment by bfallonf** [comment]
@ahardin-rh A couple nits, but looks good otherwise.

**Comment by ahardin-rh** [comment]
[rev_history]
|link:../install_config/persistent_storage/storage_troubleshooting.html[Persistent Storage Troubleshooting]
|New topic on resolving common issues encountered when configuring your cluster to use persistent storage.
%
|link:../install_config/storage_examples/index.html[Persistent Storage Examples]
|New topic on setting up and configuring common storage use cases.
%
|link:../install_config/storage_examples/shared_storage.html[Sharing an NFS Persistent Volume (PV) Across Two Pods]
|New topic on how a user wanting to leverage shared storage for use by two separate containers would configure the solution.
%

**Pull request merged**